core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -477,6 +478,22 @@ public void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountPOptions opti
     }
   }
 
+  @Override
+  public void persist(final AlluxioURI uri) throws IOException {
+    persist(uri, ScheduleAsyncPersistencePOptions.getDefaultInstance());
+  }
+
+  @Override
+  public void persist(final AlluxioURI uri, ScheduleAsyncPersistencePOptions options)
+      throws IOException {
+    FileSystemMasterClient client = mFsContext.acquireMasterClient();
+    try {
+      client.scheduleAsyncPersist(uri, options);
```

Comment:
Should we have similar exception handling as the other methods? And name `client` -> `masterClient`, `uri` -> `path`?

Reply:
something like `LOG.debug("scheduleAsyncPersist for {}, options: {}", uri.getPath(), options);` ?

Reply:
Addressed both of these in the last commit